scripts/gen_openapi.sh

```diff
@@ -87,4 +86,5 @@ fi
 kill $VAULT_PID
 sleep 1
 
-printf "\nopenapi.json generated."
```

Comment:
My shellcheck didn't pick this up. Just curious if you're not supposed to use printf.

Reply:
`printf` prints an extra character at the end, echo seemed better.